public/app/features/explore/TableContainer.tsx

```diff
@@ -71,7 +71,13 @@ export class TableContainer extends PureComponent<Props> {
       // differently and sidestep this getLinks API on a dataframe
       for (const field of dataFrame.fields) {
         field.getLinks = (config: ValueLinkConfig) => {
-          return getFieldLinksForExplore({ field, rowIndex: config.valueRowIndex!, splitOpenFn: splitOpen, range });
+          return getFieldLinksForExplore({
+            field,
+            rowIndex: config.valueRowIndex!,
+            splitOpenFn: splitOpen,
+            range,
+            dataFrame: dataFrame!,
```

Comment:
Making the entire dataframe available to internal links in the explore table. This is needed so that on the search plugin side, I can build a datalink to traceId + spanId. IE:

```
links: [{
    title: 'View Span',
    url: '',
    internal: {
      datasourceName: tempoDataSource.name,
      datasourceUid: tempoDataSource.uid,
      query: {
        queryType: 'traceId',
        query: '${__data.fields.traceID}',
      },
      // @ts-ignore
      panelsState: {
        trace: {
          spanId: '${__data.fields.spanID}',
        },
      },
    },
  },]
```